### PR TITLE
Mendix Cloud Deploy API V3

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/deploy-api-3.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Each request must contain an `Authorization` header with the value `MxToken {GENERATED_PAT}`. For example:
+
+```http {linenos=false}
+GET /api/v3/clusters HTTP/1.1
+Authorization: MxToken 7LJE…vk
+```
+
+To authenticate calls when using the Open API specification below, click **Authorize** and use the value `MxToken {GENERATED_PAT}`.
+
+## 3 Examples
```
Comment (MarkvanMents) [inline content/en/docs/apidocs-mxsdk/apidocs/deploy-api-3.md:47]: @SwapnilThange - the idea behind these examples is to show how the API calls fit within the flow of the app which is using them. And how to string several API calls together to perform a task.
The examples in the OpenUI yaml just show how to make a single call.
I could change these examples, or add calls from the V1 API to get the appId first before using it in the API calls. These were just my first attempts at showing customers how to use the API in a business context.

File: static/openapi-spec/cloud-portal-v3.yaml [hunk 2]
```diff
@@ -0,0 +1,878 @@
+openapi: 3.0.0
```
Comment (MarkvanMents) [inline static/openapi-spec/cloud-portal-v3.yaml:1]: Need an updated version of this containing before publication.

File: content/en/docs/apidocs-mxsdk/apidocs/deploy-api-3.md (hunk 1 above)
Comment (SwapnilThange) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api-3.md:47]: @MarkvanMents yes for now we can point it to v1 APIs to get the required data. As I discussed with the team today, we will soon release the v3 version of get /apps and get /apps/{appId}, then we can update the documentation again!

File: static/openapi-spec/cloud-portal-v3.yaml (hunk 2 above)
Comment (SwapnilThange) [reply static/openapi-spec/cloud-portal-v3.yaml:1]: Hi @MarkvanMents do you mean you need file only with the API's that we are going live with?

Comment (MarkvanMents) [reply static/openapi-spec/cloud-portal-v3.yaml:1]: Yes please. And with any updates you implement following my review.
I have edited a version manually to remove the unwanted ones, but I would rather you do it in whatever tool you are using to make sure I don't take out too much or leave something in which shouldn't be there (particularly the schemas).

Comment (SwapnilThange) [reply static/openapi-spec/cloud-portal-v3.yaml:1]: okay i will update and share it with you. If you need tool information its here https://gitlab.rnd.mendix.com/deployment/public-api-v3#readme , you can use the instruction for setting up

Comment (SwapnilThange) [reply static/openapi-spec/cloud-portal-v3.yaml:1]: @MarkvanMents shared an updated yaml file over email

Comment (MarkvanMents) [reply static/openapi-spec/cloud-portal-v3.yaml:1]: Thanks - I've responded by email.

**Comment by MarkvanMents** [comment]
Ready to be published.

**Comment by SwapnilThange** [comment]
Hi @MarkvanMents we are ready and want to publish this. Where shall we include the release notes?

**Comment by MarkvanMents** [comment]
Release notes can go in the Mendix Cloud release notes - I've pushed a change to this branch for you to review.

**Comment by MarkvanMents** [comment]
Publication approved over Slack

**Pull request merged**